media/videostitcher/get_slate.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stitcher
+
+// [START video_stitcher_get_slate]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	stitcher "cloud.google.com/go/video/stitcher/apiv1"
+	stitcherstreampb "google.golang.org/genproto/googleapis/cloud/video/stitcher/v1"
+)
+
+// getSlate gets a previously-created slate.
+func getSlate(w io.Writer, projectID, location, slateID string) error {
+	// projectID := "my-project-id"
+	// location := "us-central1"
+	// slateID := "my-slate-id"
+	ctx := context.Background()
+	client, err := stitcher.NewVideoStitcherClient(ctx)
+	if err != nil {
+		return fmt.Errorf("NewVideoStitcherClient: %v", err)
+	}
+	defer client.Close()
+
+	req := &stitcherstreampb.GetSlateRequest{
+		Name: fmt.Sprintf("projects/%s/locations/%s/slates/%s", projectID, location, slateID),
+	}
+
+	response, err := client.GetSlate(ctx, req)
+	if err != nil {
+		return fmt.Errorf("GetSlate: %v", err)
+	}
+
+	fmt.Fprintf(w, "Slate: %v", response.Name)
```

Comment:
https://googlecloudplatform.github.io/samples-style-guide/#result  Would be good to print the details of the slate that you retrieved as well.

Reply:
Done